test/parser/testdata.py

```diff
@@ -1,19 +1,25 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright (c) 2017, the cclib development team
+# Copyright (c) 2019, the cclib development team
 #
 # This file is part of cclib (http://cclib.github.io) and is distributed under
 # the terms of the BSD 3-Clause License.
 
 """Unit tests for parser data module."""
 
+import mock
 import unittest
 
 import numpy
 
 import cclib
 
 
+class AnyStringWith(str):
```

Comment:
I understand exactly what this does from reading `__eq__`, but not the class name. Can you change it to something containing `Contains` or similar?

Reply:
Changed to `StringContains`, hopefully that's good.